examples/vision/3D_image_classification.py

```diff
@@ -0,0 +1,482 @@
+"""
+Title: 3D Image Classification from CT Scans
+Author: [Hasib Zunair](https://twitter.com/hasibzunair)
+Date created: 2020/09/23
+Last modified: 2020/09/23
+Description: Train a 3D convolutional neural network to predict presence of pneumonia.
+"""
+"""
+## Introduction
+
+This example will show the steps needed to build a 3D convolutional neural network (CNN)
+to predict the presence of viral pneumonia in computer tomography (CT) scans.
+
+2D CNNs are commonly used to process RGB images (3 channels). A 3D CNN is simply the 3D
+equivalent:
+it takes as input a 3D volume or a sequence of 2D frames (e.g. slices in a CT scan),
+3D CNNs are a powerful model for learning representations for volumetric data.
+
+## References
+
+- [A survey on Deep Learning Advances on Different 3D
+DataRepresentations](https://arxiv.org/pdf/1808.01462.pdf)
+- [VoxNet: A 3D Convolutional Neural Network for Real-Time
+ObjectRecognition](https://www.ri.cmu.edu/pub_files/2015/9/voxnet_maturana_scherer_iros15.pdf)
+ObjectRecognition](https://www.ri.cmu.edu/pub_files/2015/9/voxnet_maturana_scherer_iros15.pdf)
+- [FusionNet: 3D Object Classification Using MultipleData
+Representations](http://3ddl.cs.princeton.edu/2016/papers/Hegde_Zadeh.pdf)
+- [Uniformizing Techniques to Process CT scans with 3D CNNs for Tuberculosis
+Prediction](https://arxiv.org/abs/2007.13224)
+"""
+
+import os
+import zipfile
+import numpy as np
+import tensorflow as tf
+
+from tensorflow import keras
+from tensorflow.keras import layers
+
+"""
+## Downloading the MosMedData:Chest CT Scans with COVID-19 Related Findings
+
+In this example, we use a subset of the
+[MosMedData: Chest CT Scans with COVID-19 Related
+Findings](https://www.medrxiv.org/content/10.1101/2020.05.20.20100362v1). This dataset
+consists of lung CT scans with COVID-19 related findings, as well as without such
+findings.
+
+We will be using the associated radiological findings of the CT scans as labels to build
+a classifier to predict presence of viral pneumonia.
+Hence, the task is a binary classification problem.
+"""
+
+# download url of normal CT scans
```

Comment:
One last nit: make sure all comments in the file start with a capital and end with a period.

Reply:
Could you please check before I add the generated files?